index.html

```diff
@@ -182,6 +182,25 @@ <h2> About <br> Myself</h2>
             </ul>
           </div>
           <hr>
+
+          <section class="section-form">
+            <div class="form-div">
+             <h3>Contact me</h3>
+             <p>If you have an application you are 
+               interested in developing,
+               a feature that you need built or a project that 
+               needs coding. I’d love to help with it </p>
+            </div>
+            <form action="https://formspree.io/f/{form_id}" method="post">
+               <ul>
+              <li><input type="text" id="name" name="user_name" placeholder="Write your name here" maxlength="30" required></li>
+              <li><input type="email" id="mail" name="user_email" placeholder="Write your email here" required></li>
+              <li><textarea type="text" id="msg" name="user_message" rows="8" cols="50" placeholder="write your message here" maxlength="500" required></textarea></li>
+             <li><button type="submit" class="form-button">Get in touch</button></li>
```

Comment:
- [ ] [OPTIONAL] I have noticed that you have a scroll on the mobile version, particularly the contact section. The experience of scrolling horizontally can make the screen judder and create a headache-inducing feeling. Kindly ensure that there is no horizontal scroll.

- Your design 😎 
![scroll](https://user-images.githubusercontent.com/99394374/214550536-d42b5c24-3faa-44b4-86df-546d2576e07c.PNG)